src/app/header/components/TermButtons.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
+import React, { useCallback, useState } from 'react';
+
+import { useTerm } from '../../context/TermContext';
+import { getCurrentTerms, getReadableTerm } from '../../shared/utils/terms';
+
+export function TermButtons(): JSX.Element {
+  const [, setTerm] = useTerm();
+  const [status, setStatus] = useState([true, false, false]);
+
+  const terms = getCurrentTerms();
```

Comment:
I feel all three can be combined into a utility hook called `useTerm`. It could maybe have the follow return attributes like `currentTerm, prevTerm, nextTerm, setTerm` etc. The `currentTerm` for example could be structured like `selected: true, formatted: '202101'` etc?